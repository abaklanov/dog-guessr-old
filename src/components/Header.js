import React from "react";
import { Navbar } from "react-bootstrap";

class Header extends React.Component {
  render() {
    return (
      <Navbar bg="light">
        <Navbar.Brand href="/">DogGuessr</Navbar.Brand>
      </Navbar>
    );
  }
}

export default Header;
