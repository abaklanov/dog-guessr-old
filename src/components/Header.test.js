import React from "react";
import { shallow } from "enzyme";
import Header from "./Header";

const wrapper = shallow(<Header />);

it("renders correctly", () => {
  expect(wrapper).toMatchSnapshot();
});
