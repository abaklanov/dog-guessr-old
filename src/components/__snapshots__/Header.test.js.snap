// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<ForwardRef(Bootstrap(undefined))
  bg="light"
>
  <ForwardRef(Bootstrap(NavbarBrand))
    href="/"
  >
    DogGuessr
  </ForwardRef(Bootstrap(NavbarBrand))>
</ForwardRef(Bootstrap(undefined))>
`;
